lorien/UI/Dialogs/KeyBindingsList.gd

```diff
@@ -0,0 +1,66 @@
+extends Node
+
+# -------------------------------------------------------------------------------------------------
+const KEYBINDINGS_LINE_SCENE = preload("res://UI/Components/KeyBindingsLine.tscn")
+
+# -------------------------------------------------------------------------------------------------
+onready var _GRID := $ScrollContainer/KeyBindingsList
+onready var _ADD_KEY_DIALOG := $AddKeyDialog
```

Comment:
why are these uppercase?

Reply:
Well... style question: These _are_ pretty much constants, but because of onready they need to be var, I think... my guess was that the case should reflect _intent_. Admittedly, I checked the codebase and I think all other code is doing it lowercase... so I am going to fix it :+1:

Reply:
In my mind if objects are mutable (meaning you can change values inside them either directly or indirectly) they are not constant...doesn't matter how you use them. For example you use `_GRID.add_child(child)`. But yeah, for constants the uppercase version would be correct :)

Reply:
Yeah. You see my Python background shining though here. `const` is not a thing over there, and you use case to signal intent. Anyway, does not really matter. lowercase it is :-)